### PR TITLE
i18n(zh-cn): update cli-reference

### PR DESCRIPTION
**Pull request by liruifengv**
#### What kind of changes does this PR include?

- Translated content
#### Description

- update cli-reference

### CODE REVIEW

File: src/content/docs/zh-cn/reference/cli-reference.mdx [hunk 1]
```diff
@@ -78,32 +152,38 @@ astro telemetry clear
...
-根目录是用来寻找 Astro 配置文件的。
+指定配置文件的路径。默认为 `astro.config.mjs`。如果使用了不同的配置文件名称或放在其他文件夹中，请使用：
 
 ```shell
-astro --root myRootFolder/myProjectFolder dev
+astro --config config/astro.config.mjs dev
 ```
 
-### `--reload`
+### `--site`
```
Comment (liruifengv) [inline src/content/docs/zh-cn/reference/cli-reference.mdx:178]:
```suggestion
### `--site  <url>`
```

File: src/content/docs/zh-cn/reference/cli-reference.mdx [hunk 2]
```diff
@@ -78,32 +152,38 @@ astro telemetry clear
...
+
+为您的项目配置[`site`值](/zh-cn/reference/configuration-reference/#base)。 传递此标志将覆盖 `astro.config.mjs` 文件中的`site`值（如果存在）。
+
+### `--base`
```
Comment (liruifengv) [inline src/content/docs/zh-cn/reference/cli-reference.mdx:182]:
```suggestion
### `--base <pathname>`
```

**Review by delucis — APPROVED** [review]
This looks good to me! Thank you @liruifengv 🙌

File: src/content/docs/zh-cn/reference/cli-reference.mdx [hunk 3]
```diff
@@ -47,17 +109,29 @@ title: CLI 参考
...
-### `astro add`
+## `astro sync`
+
+<p>
+  <Since v="2.0.0" />
+</p>
 
-在配置中添加集成
+:::tip
+运行 `astro dev` 或者 `astro build` 也会运行 `sync`命令。
```
Comment (liruifengv) [inline src/content/docs/zh-cn/reference/cli-reference.mdx:119]:
```suggestion
运行 `astro dev`，`astro build`或者`astro check`也会运行`sync`命令。
```

**Comment by Yan-Thomas** [comment]
LGTM! Thanks @liruifengv 💜

**Pull request merged**